src/ScriptAwareTrait.php

```diff
@@ -11,19 +11,30 @@
 
 namespace OpenSearchDSL;
 
+use OpenSearchDSL\Aggregation\Metric\StatsAggregation;
+
 /**
  * A trait which handles elasticsearch aggregation script.
  */
 trait ScriptAwareTrait
 {
-    private ?string $script = null;
+    /**
+     * @var string|array{id: string, params?: array<string, mixed>}|null
```

Comment:
Seems like type aliases don't work in traits, therefore i copy&pasted the array shape for now